src/rpc/governance.cpp

```diff
@@ -273,7 +273,7 @@ UniValue gobject_submit(const JSONRPCRequest& request)
 
     if(govobj.GetObjectType() == GOVERNANCE_OBJECT_PROPOSAL) {
         CProposalValidator validator(strDataHex);
-        if(!validator.Validate())  {
+        if(!validator.Validate()) {
```

Comment:
Can you add a space after the `if` here since it's a formatting change touching the same line?